ergo-wallet/src/main/scala/org/ergoplatform/wallet/boxes/DefaultBoxSelector.scala

```diff
@@ -85,42 +87,32 @@ object DefaultBoxSelector extends BoxSelector {
     }
   }
 
-  def formChangeBoxes(
-                       foundBalance: Long,
-                       targetBalance: Long,
-                       foundBoxAssets: mutable.Map[ModifierId, Long],
-                       targetBoxAssets: TokensMap
-                     ): Either[BoxSelectionError, Seq[ErgoBoxAssets]] = {
-    AssetUtils.subtractAssetsMut(foundBoxAssets, targetBoxAssets)
-    val changeBoxesAssets: Seq[mutable.Map[ModifierId, Long]] = foundBoxAssets.grouped(MaxTokens).toSeq
+  def formChangeBox(
+                     foundBalance: Long,
+                     targetBalance: Long,
+                     foundBoxAssets: mutable.Map[ModifierId, Long],
+                     targetBoxAssets: TokensMap
+                   ): Either[BoxSelectionError, Option[ErgoBoxAssets]] = {
```

Comment:
better move refactoring to another PR, make ScalaDoc for the method, and logic descriptions in comments within the method